### PR TITLE
normalize comment on DeleteRelease function

### LINKED ISSUE #362: [DOC]Normalization of comments on func DeleteRelease(pkg/utils/helm/utils.go)
**Provide a link to that doc page:**
https://github.com/fluid-cloudnative/fluid/blob/master/pkg/utils/helm/utils.go

**What is the defect and your suggestions on improvement:**
Annotation is not standardized. I will standardize the comments on func DeleteRelease.

### PR DESCRIPTION
**Pull request by SunYiXuan98**
Ⅰ. Describe what this PR does
I normalized the comment on pkg/utils/helm/utils.go:DeleteRelease function.

Ⅱ. Does this pull request fix one issue?
fix #362 

Ⅲ. List the added test cases (unit test/integration test) if any, please explain if no tests are needed.
These is no test cases because this task just modifies a comment, and does not do anything on original codes.

Ⅳ. Describe how to verify it
Verify the comments with stardard Go's Comment Standard.

Ⅴ. Special notes for reviews
None.

### CODE REVIEW

**Comment by cheyang** [comment]
Please open an issue to fix, thanks.

**Comment by SunYiXuan98** [comment]
I opened on issue on #362 .

File: pkg/utils/helm/utils.go [hunk 1]
```diff
@@ -115,9 +115,7 @@ func CheckRelease(name, namespace string) (exist bool, err error) {
 	return exist, err
 }
 
-/*
-* Delete Release given name and namespace
- */
+// DeleteRelease given name and namespace
```
Comment (cheyang) [inline pkg/utils/helm/utils.go:118]: DeleteRelease deletes release  with the name and namespace

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/utils/helm/utils.go (hunk 1 above)
Comment (SunYiXuan98) [reply pkg/utils/helm/utils.go]: Thanks,and I changed the comment just now.

**Review by cheyang — APPROVED** [review]

**Pull request merged**